src/main/java/lotto/service/LottoService.java

```diff
@@ -28,21 +26,32 @@ public LottoService(Money money) {
 
 
     public Lottos buyLotto() {
-        Money money = wallet.getMoneyToBuy();
-        Lottos lottos = lottoSeller.buyLotto(money);
-        wallet.saveLottos(lottos);
-        return lottos;
+        return wallet.buyLotto(lottoSeller);
+    }
+
+    public WinningHistory getWinningHistory(Lotto winningLotto, LottoNumber bonus) {
+        Lottos lottos = wallet.getLottos();
+        List<WinningRank> winningRanks = checkWinning(lottos, winningLotto, bonus);
+        Money winningMoney = getWinningMoney(winningRanks);
+        Money originMoney = wallet.getMoneyToBuy();
+
+        return WinningHistory.create(originMoney, winningRanks, winningMoney);
+    }
+
+    private List<WinningRank> checkWinning(Lottos lottos, Lotto winningLotto, LottoNumber bonus) {
```

Comment:
```suggestion
    private List<WinningRank> getWinnings(Lottos lottos, Lotto winningLotto, LottoNumber bonus) {
```
컬렉션을 반환하니 이런 느낌의 이름이 좋을 것 같아요 :)